components/automate-ui/src/index.html

```diff
@@ -15,6 +15,11 @@
     !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on","addSourceMiddleware"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="4.0.0";
     }}();
   </script>
+  <script type="text/javascript">
+    (function (c, h, e, f) {
+      let y, z;if(c && h && e && f){y = h.createElement(e);y.async = !0;y.src = 'https://d2mopladttg8fn.cloudfront.net/chef-telemetry-tracker.js?time='+new Date().getTime();z = h.getElementsByTagName(f)[0];z.appendChild(y);}
```

Comment:
we need to move this url to env variable as the current one is pointing to staging. Please make the env changes while I try configure the prod url for the same.

Reply:
Updated the production url as we are not going with env based approach.